inference-engine/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/fake_quantize.cpp

```diff
@@ -50,7 +50,7 @@ const std::vector<std::vector<size_t>> inputShapes = {
             {8}
             };
 const std::vector<std::vector<size_t>> constShapes = {{1}};
-const std::vector<size_t> levels = {16, 255, 256};
+const std::vector<size_t> levels = {16, 255, 256, 4294967296};
```

Comment:
Maybe it is better to use SIZE_MAX here?

Reply:
done